src/components/Trackpad/ControlBar.tsx

```
@@ -1,51 +1,51 @@
-                onPointerDown={(e) => handleInteraction(e, onKeyboardToggle)}
-            >
-                Keyboard
-            </button>
-        </div>
-    );
+	return (
+		<div className="bg-base-200 p-2 grid grid-cols-4 gap-2 shrink-0">
+			<button
+				className={`btn btn-sm ${scrollMode ? "btn-primary" : "btn-outline"}`}
+				onPointerDown={(e) => handleInteraction(e, onToggleScroll)}
+			>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)